backend/app/store/image/image.go

```diff
@@ -186,17 +175,18 @@ func (s *Service) ExtractPictures(commentHTML string) (ids []string, err error)
 	return ids, nil
 }
 
-// Cleanup runs periodic cleanup with cleanupTTL. Blocking loop, should be called inside of goroutine by consumer
+// Cleanup runs periodic cleanup with 2.5*ServiceParams.EditDuration. Blocking loop, should be called inside of goroutine by consumer
 func (s *Service) Cleanup(ctx context.Context) {
-	log.Printf("[INFO] start pictures cleanup, staging ttl=%v", s.cleanupTTL)
+	cleanupTTL := s.EditDuration * 25 / 10 // cleanup images older than 1.5 * EditDuration
```

Comment:
well, 25/10 is 2.5 not 1.5. Maybe just do direct *1.5 (or *2.5)?

Reply:
Fixed. There is type confusion otherwise, that is a way to keep numbers straight without types conversions.